package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@microsoft/azure-iot-ux-fluent-controls",
-    "version": "6.0.0-uialignment.1",
+    "version": "6.0.0-uialignment.3",
```

Comment:
add an entry to the CHANGELOG as well

Reply:
I will update the changelog when will move to 6.0.0